File: Office4U.Articles.ImportExport.Api/Entities/Article.cs
namespace Api.Entities
{
    public class Article
    {
        public int Id { get; set; }
        public string Code { get; set; }
        public string SupplierId { get; set; }
        public string SupplierReference { get; set; }
        public string Name1 { get; set; }
        public string Unit { get; set; }
        public decimal PurchasePrice { get; set; }
    }
}

File: Office4U.Articles.ImportExport.Web/src/app/_models/article.ts
import { ArticlePhoto } from "./articlePhoto";

export interface Article {
    id: number;
    code: string;
    supplierId: string;
    supplierReference: string;
    name1: string;
    unit: string;
    purchasePrice: number;
    photoUrl: string;
    photos: Array<ArticlePhoto>;
}


